hw/dv/tools/xcelium/cover.ccf

```diff
@@ -27,7 +27,8 @@ deselect_coverage -betfs -module prim_cdc_rand_delay
 deselect_coverage -betf -module *csr_assert_fpv...
 
 // Only collect toggle coverage on the DUT and the black-boxed IP (above) ports.
-
+deselect_coverage -toggle -module ${DUT_TOP}...
+select_coverage -toggle -module ${DUT_TOP}
```

Comment:
@rasmus-madsen 
@cindychip and I found all level's toggle coverages are enabled now, after removing these 2 lines in https://github.com/lowRISC/opentitan/pull/12400.
LMK if you recall any particular reason to remove them. Otherwise, I feel we need these 2 lines.

Reply:
oh that is my bad - I was cleaning up and I must have missed that we collected port on all everything after removing this.
adding this is ok with me.